packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -33,8 +36,11 @@ export class NodeHttpRequestResponseHandler implements NodeHttpStreamsHandler {
     switch (contentType) {
 
       case 'application/json':
+        this.logger.info('Parsing body as JSON', contentType);
+
         return JSON.parse(body);
       // case 'application/x-www-form-urlencoded':
+      //   this.logger.info('Parsing body as form data', contentType);
```

Comment:
Could possibly log body, but that would also log a lot lot.
```suggestion
        this.logger.info('Parsing body as JSON');

        return JSON.parse(body);
      // case 'application/x-www-form-urlencoded':
      //   this.logger.info('Parsing body as form data');
```